cmd/sealer/cmd/config.go

```diff
@@ -0,0 +1,78 @@
+// Copyright © 2023 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+func DefaultConfig() *Config {
+	bc := BuildConfig{
+		Compressed:   false,
+		RegistryType: "docker",
+	}
+
+	return &Config{
+		DebugOn:   false,
+		LogToFile: true,
+		ColorMode: "always",
+		Cluster: ClusterConfig{
+			CacheImage: false,
+			Prune:      true,
+		},
+		Image: ImageConfig{
+			Build: bc,
+		},
+	}
+}
+
+type Config struct {
+	// Debug refers to the log mode.
+	DebugOn bool `json:"debugOn"`
+
+	LogToFile bool `json:"logToFile"`
+
+	RemoteLoggerURL string `json:"remoteLoggerURL"`
+
+	RemoteLoggerTaskName string `json:"remoteLoggerTaskName"`
+
+	ColorMode string `json:"colorMode"`
+
+	Image ImageConfig `json:"image"`
+
+	Cluster ClusterConfig `json:"cluster"`
+}
+
+type ImageConfig struct {
+	Build BuildConfig `json:"build"`
+}
+
+type BuildConfig struct {
+	//docker: use docker registry data format
+	//oci: use oci registry data format
+	RegistryType string `json:"registryType"`
+
+	// whether to compress registry.
+	// default is false.
+	Compressed bool `json:"compressed"`
+}
+
+type ClusterConfig struct {
```

Comment:
I wonder if using `clusterConfig` is appropriate, it would make me think at first that kxs runtime has something to do with it, but it doesn't seem to be